docs/GettingStarted.md

```diff
@@ -2313,3 +2314,70 @@ However, additional instrumentation provided by Datadog can be activated alongsi
 | `OpenTracing::FORMAT_TEXT_MAP` | Yes        |                        |
 | `OpenTracing::FORMAT_RACK`     | Yes        | Because of the loss of resolution in the Rack format, please note that baggage items with names containing either upper case characters or `-` will be converted to lower case and `_` in a round-trip respectively. We recommend avoiding these characters or accommodating accordingly on the receiving end. |
 | `OpenTracing::FORMAT_BINARY`   | No         |                        |
+
+### Profiling
+
+*Currently available as BETA feature.*
+
+`ddtrace` can produce profiles that measure method-level application resource usage within production environments. These profiles can give insight into resources spent in Ruby code outside of existing trace instrumentation.
+
+**Compatibility**
+
+Basic profiling requires:
+
+- MRI Ruby 2.0+
+- `google-protobuf` gem installed
+
+In addition, CPU time measurements require:
+
+- MRI Ruby 2.1+
+- Linux OS with `pthread` support
```

Comment:
There's a fallback method that we should implement for other OSes:
https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/profiling/collector/stack.pyx#L127-L148

It boils down to doing `process_time / nb_threads` which is not completely accurate, but still gives a rough idea of what's going on.

Reply:
We can do this, but we should do this in another PR.